butterknife-compiler/src/main/java/butterknife/compiler/FieldDrawableBinding.java

```diff
@@ -25,7 +25,7 @@
   }
 
   @Override public CodeBlock render(int sdk) {
-    if (tintAttributeId.value != 0) {
+    if (tintAttributeId.value != -1) {
```

Comment:
this `-1` is kind of a magic number. it's not clear to me from looking at this class why the value would ever be -1.
perhaps make a `NO_ID` constant in `BindDrawable`?

Reply:
Was thinking about the same. But BindDrawable may not be the best place for it